src/axom/quest/readers/C2CReader.cpp

```diff
@@ -377,5 +888,247 @@ void C2CReader::getLinearMesh(mint::UnstructuredMesh<mint::SINGLE_SHAPE>* mesh,
   }    // end for each NURBS curve
 }
 
+//---------------------------------------------------------------------------
+void C2CReader::getLinearMesh(mint::UnstructuredMesh<mint::SINGLE_SHAPE>* mesh,
+                              double percentError)
```

Comment:
Suggestion:
Should we let the user specify an optional third parameter for the maximum number of segments to prevent the code from running out of memory for really small `percentError`. (Perhaps setting this to 0 or a negative value would allow it to be unbounded.)

We can either expose it as a third parameter to this function, or as another optional parameter function that can be called before `getLinearMesh`, e.g. `C2CReader::setMaxLinearSegments(int)`

Reply:
There could be a parameter. It's already capped at 2000 internally. There was so little improvement in the volume beyond that, at least for circles.